syntax = "proto2";

package foundation;

message ComparedValue {
  required string id = 1;
  required string target_id = 2;
  required string extracted_id = 3;
  optional float score = 4;
  optional string message = 5;
}

message ComparedValueCollection {
  required string id = 1;
  required string compared_value_ids = 2;
}